torch/csrc/Exceptions.h

```diff
@@ -68,4 +68,6 @@ struct python_error : public std::exception {
 bool THPException_init(PyObject *module);
 #endif
 
+const char * processErrorMsg(std::string str);
```

Comment:
Put this in a `namespace torch` or prefix it with `THPException_`. (Namespaces are preferable).